src/test/docker/bionic/Dockerfile

```diff
@@ -18,7 +18,10 @@ RUN \
  fi
 
 RUN \
- sudo python3.7 -m pip install pandas==0.24.1
+ sudo pip3 install pandas==0.24.1
+
+ENV \
+ PYTHONPATH=$PYTHONPATH:/usr/lib/python3.6/site-packages/
```

Comment:
Is this `ENV` line really necessary (Shouldn't `/usr/lib/python3.6/site-packages` be in the default `PYTHONPATH` on bionic?)

Reply:
If I leave this line out, I get the following `ModuleNotFoundError` after running `sudo make install`:

```console
moussa1@docker-desktop:/usr/src$ flux account -h
Traceback (most recent call last):
  File "/usr/libexec/flux/cmd/flux-account.py", line 15, in <module>
    import fluxacct.accounting
ModuleNotFoundError: No module named 'fluxacct'
``` 

even though `fluxacct/accounting/` gets installed here:

```console
moussa1@docker-desktop:/usr/src$ ls -l /usr/lib/python3.6/site-packages/
total 12
drwxr-xr-x 3 root root 4096 Feb 25 16:33 _flux
drwxr-xr-x 6 root root 4096 Feb 25 16:33 flux
drwxr-xr-x 3 root root 4096 Apr  8 15:29 fluxacct
```

And if I `echo $PYTHONPATH` I don't seem to get anything:

```console
moussa1@docker-desktop:/usr/src$ echo $PYTHONPATH

moussa1@docker-desktop:/usr/src$
```

Only when I explicitly add that `site-packages` path to my `PYTHONPATH` do I get flux-accounting output. Maybe I am messing up somewhere else?

Reply:
I think you can see python's default path by something like:
```
python3 -c 'import sys; print(sys.path)'
```

Reply:
Ah, okay. Yes, when I run this, I see `/usr/lib/python3.6/site-packages`:

```console
moussa1@docker-desktop:/usr/src$ python3 -c 'import sys; print(sys.path)'
['', '/usr/src', '/usr/lib/python3.6/site-packages', '/usr/lib/python36.zip', '/usr/lib/python3.6', '/usr/lib/python3.6/lib-dynload', '/usr/local/lib/python3.6/dist-packages', '/usr/lib/python3/dist-packages']
```

Interesting. I can't imagine the default version of `python` on `bionic` would affect anything right? 

```console
moussa1@docker-desktop:/usr/src$ python --version
Python 2.7.17
```

Since the version of `flux python` is `3.6`:

```console
moussa1@docker-desktop:/usr/src$ flux python --version
Python 3.6.9
```

---

FWIW, the default version of Python on `centos8`:

```console
[moussa1@docker-desktop src]$ python --version
Python 3.6.8
```

Reply:
I think the problem here is that the python packages are being installed to `site-packages` whilst ubuntu bionic wants/expects them to be installed in `dist-packages`, which also affects flux-core and is being tracked here: https://github.com/flux-framework/flux-core/issues/2617.  If so, I think we will need some autotools magic to make this work.

Reply:
Sorry, it is more subtle than my original post.  It is true that if things were installed in `dist-packages`, the module would be found, but flux-accounting is also missing the symlink thingy that we do in other projects so that the `flux` starter can safely add a path to PYTHONPATH without also pulling in a bunch of system modules:
```
herbein1@docker-desktop:/usr/src$ printenv PYTHONPATH
:/usr/lib/python3.6/site-packages/
herbein1@docker-desktop:/usr/src$ unset PYTHONPATH
herbein1@docker-desktop:/usr/src$ printenv PYTHONPATH
herbein1@docker-desktop:/usr/src$ python3 -c 'import sys; print(sys.path)'
['', '/usr/lib/python36.zip', '/usr/lib/python3.6', '/usr/lib/python3.6/lib-dynload', '/usr/local/lib/python3.6/dist-packages', '/usr/lib/python3/dist-packages']
herbein1@docker-desktop:/usr/src$ flux python -c 'import sys; print(sys.path)'
['', '/usr/lib/flux/python3.6', '/usr/lib/python36.zip', '/usr/lib/python3.6', '/usr/lib/python3.6/lib-dynload', '/usr/local/lib/python3.6/dist-packages', '/usr/lib/python3/dist-packages']
herbein1@docker-desktop:/usr/src$ ls /usr/lib/flux/python3.6
_flux  flux
herbein1@docker-desktop:/usr/src$ ls -la /usr/lib/flux/python3.6/
total 8
drwxr-xr-x 2 root root 4096 Feb 26 23:57 .
drwxr-xr-x 5 root root 4096 Feb 26 23:57 ..
lrwxrwxrwx 1 root root   38 Feb 26 23:57 _flux -> /usr/lib/python3.6/site-packages/_flux
lrwxrwxrwx 1 root root   37 Feb 26 23:57 flux -> /usr/lib/python3.6/site-packages/flux
```

So if we copy over that symlink thing from flux-core autotools, things should start working again in flux-accounting and we can punt on fixing #2617 for now.

Reply:
Thanks for your help @SteVwonder. I was able to add the autotools support (copied over from flux-core) this morning and have this symlink be generated when flux-accounting is built and installed. Here it is shown on `bionic`:

```console
moussa1@docker-desktop:/usr/src$ ls -l /usr/lib/flux/python3.6/
total 0
lrwxrwxrwx 1 root root 38 Feb 25 16:33 _flux -> /usr/lib/python3.6/site-packages/_flux
lrwxrwxrwx 1 root root 37 Feb 25 16:33 flux -> /usr/lib/python3.6/site-packages/flux
lrwxrwxrwx 1 root root 41 Apr  9 14:50 fluxacct -> /usr/lib/python3.6/site-packages/fluxacct
```

And I can run `flux account` commands after running `sudo make install` without having to manually append to my `PYTHONPATH`:

```console
moussa1@docker-desktop:/usr/src$ echo $PYTHONPATH

moussa1@docker-desktop:/usr/src$ flux account -h
usage: flux-account.py [-h] [-p PATH] [-o OUTPUT_FILE]
                       {view-user,add-user,delete-user,edit-user,view-job-records,create-db,add-bank,view-bank,delete-bank,edit-bank}
                       ...

Description: Translate command line arguments into SQLite instructions for the
Flux Accounting Database.

positional arguments:
  {view-user,add-user,delete-user,edit-user,view-job-records,create-db,add-bank,view-bank,delete-bank,edit-bank}
                        sub-command help
    view-user           view a user's information in the accounting database
    add-user            add a user to the accounting database
    delete-user         remove a user from the accounting database
    edit-user           edit a user's value
    view-job-records    view job records
    create-db           create the flux-accounting database
    add-bank            add a new bank
    view-bank           view bank information
    delete-bank         remove a bank
    edit-bank           edit a bank's allocation

optional arguments:
  -h, --help            show this help message and exit
  -p PATH, --path PATH  specify location of database file
  -o OUTPUT_FILE, --output-file OUTPUT_FILE
                        specify location of output file
```

I also built and installed on the `centos8` image to make sure it worked there as well. So I'll go ahead and remove the manual append to `PYTHONPATH` in the `bionic` image and squash the changes made to **configure.ac** and **src/bindings/python/fluxacct/Makefile**.